docs/content/docs/operators/recommendation/swing.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Swing"
+type: docs
+aliases:
+- /operators/recommendation/swing.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Swing
+
+An AlgoOperator which implements the Swing algorithm.
+
+Swing is an item recall algorithm. The topology of user-item graph usually can be described as
+user-item-user or item-user-item, which are like 'swing'. For example, if both user <em>u</em> and user <em>v</em>
+have purchased the same commodity $i$, they will form a relationship diagram similar to a swing. If
+<em>u</em> and <em>v</em> have purchased commodity <em>j</em> in addition to <em>i</em>, it is supposed <em>i</em>
+and <em>j</em> are similar. 
+
+See "<a href="https://arxiv.org/pdf/2010.05525.pdf">Large Scale Product Graph Construction for Recommendation in
+E-commerce</a>" by Xiaoyong Yang, Yadong Zhu and Yi Zhang.
+
+### Input Columns
+
+| Param name | Type | Default  | Description |
+|:-----------|:-----|:---------|:------------|
+| itemCol    | Long | `"item"` | Item id.    |
+| userCol    | Long | `"user"` | User id     |
+### Output Columns
+
+| Param name | Type   | Default        | Description                                                                 |
+|:-----------|:-------|:---------------|:----------------------------------------------------------------------------|
+| itemCol    | Long   | `"prediction"` | Item id.                                                                    |
+| outputCol  | String | `"output"`     | Recommendations and their score. (e.g. "item_1,0.9;item_2,0.7;item_3,0.35") |
```

Comment:
What about using the following description?

`Top k similar items and their corresponding scores (e.g. "item_1,0.9;item_2,0.7;item_3,0.35").`